StorageUtility/Windows/StorageUtility.cpp

```diff
@@ -116,9 +134,10 @@ eErrorCode EnumerateDevices( sEnumerateDevicesCallback& Callback, const GUID* In
 
         eErrorCode errorCode;
         errorCode = eErrorCode::Unknown;
-        Callback.Function( Callback.Data, devInfoHandle, devInfoData, pDevDetailData, errorCode );
+        Callback.Function(Callback.Data, devInfoHandle, devInfoData, devInterfaceData, pDevDetailData, errorCode);
 
-        LocalFree( pDevDetailData );
+        // TODO: need to be updated on the Issue #5: LocalAlloc and LocalFree should be replaced with heap functions.
+        //LocalFree( pDevDetailData ); // Locked this line otherwise its data will be removed
```

Comment:
This variable was kept as a part of cDevice, it shouldn't free here
It should have might free after cDevice was destructed